tools/tests/py2fgen/test_diffusion_wrapper.py

```diff
@@ -6,146 +6,528 @@
 # Please, refer to the LICENSE file in the root directory.
 # SPDX-License-Identifier: BSD-3-Clause
 
-# type: ignore
-from gt4py.next import as_field
-from icon4py.model.atmosphere.diffusion.diffusion import DiffusionType
+from unittest import mock
+
+import gt4py.next as gtx
+import numpy as np
+import pytest
+from icon4py.model.atmosphere.diffusion import diffusion
+from icon4py.model.atmosphere.diffusion.diffusion import DiffusionType, TurbulenceShearForcingType
 from icon4py.model.common import dimension as dims
-from icon4py.model.common.settings import xp
-from icon4py.model.common.test_utils.grid_utils import MCH_CH_R04B09_LEVELS
+from icon4py.model.common.constants import DEFAULT_PHYSICS_DYNAMICS_TIMESTEP_RATIO
+from icon4py.model.common.grid import vertical as v_grid
+from icon4py.model.common.grid.geometry import CellParams, EdgeParams
+from icon4py.model.common.test_utils import datatest_utils as dt_utils, helpers
+from icon4py.model.common.test_utils.datatest_utils import (
+    get_global_grid_params,
+)
+from icon4py.model.common.test_utils.diffusion_utils import (
+    construct_config,
+    construct_diagnostics,
+    construct_interpolation_state,
+    construct_metric_state,
+    vertical_grid,
+)
 
+import icon4pytools.py2fgen.wrappers.diffusion
+from icon4pytools.py2fgen.wrappers import wrapper_dimension
 from icon4pytools.py2fgen.wrappers.diffusion import diffusion_init, diffusion_run
```

Comment:
Is it possible to just use one import for ` icon4pytools.py2fgen.wrappers.diffusion`?

Reply:
Done